pkg/cvo/metrics.go

```diff
@@ -297,6 +307,18 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 		g.Set(1.0)
 		ch <- g
 	}
+
+	if availableUpdates := m.optr.getAvailableUpdates(); availableUpdates != nil {
+		klog.Infof("Status: %s", availableUpdates.Condition.Status)
+		if availableUpdates.Condition.Status == configv1.ConditionFalse {
```

Comment:
What about the "we're fine" case? Or the "unknown" case (which we may not set)?  How about something like:

```go
g := m.clusterVersionOperatorRetrieving
switch availableUpdates.Condition.Status {
case configv1.ConditionTrue:
  g.Set(1)
case configv1.ConditionFalse:
  g = g.WithLabelValues(availableUpdates.Condition.Reason, availableUpdates.Condition.Message)
  g.Set(0)
default:
  klog.Warnf("unknown update status: ...", ...)
}
ch <- g
```

Reply:
Will do. I actually wasn't done but that's okay because wasn't sure about the Set(1) since it doesn't appear to always be done in other places. However, it makes sense to set it. For example, it allows someone to simply add another dependency into the metrics yaml file based on it being true without updating the code.

Reply:
if there is some established precedent you are following where we don't push a `Set(1)` version, then I'm fine following that precedent.  I just thought our other metrics were always set, not just set when there were problems.